test/dump/try-delegate.txt

```diff
@@ -0,0 +1,99 @@
+0000014: 01                                        ; num functions
+0000015: 01                                        ; function 0 signature index
+0000013: 02                                        ; FIXUP section size
+; section "Event" (13)
+0000016: 0d                                        ; section code
+0000017: 00                                        ; section size (guess)
+0000018: 01                                        ; event count
+; event 0
+0000019: 00                                        ; event attribute
+000001a: 00                                        ; event signature index
+0000017: 03                                        ; FIXUP section size
+; section "DataCount" (12)
+000001b: 0c                                        ; section code
+000001c: 00                                        ; section size (guess)
+000001d: 00                                        ; data count
+000001c: 01                                        ; FIXUP section size
+; section "Code" (10)
+000001e: 0a                                        ; section code
+000001f: 00                                        ; section size (guess)
+0000020: 01                                        ; num functions
+; function body 0
+0000021: 00                                        ; func body size (guess)
+0000022: 00                                        ; local decl count
+0000023: 06                                        ; try
+0000024: 7f                                        ; i32
+0000025: 06                                        ; try
+0000026: 40                                        ; void
+0000027: 01                                        ; nop
+0000028: 18                                        ; delegate
+0000029: 01                                        ; delegate depth
+000002a: 41                                        ; i32.const
+000002b: 07                                        ; i32 literal
+000002c: 07                                        ; catch
+000002d: 00                                        ; catch event
+000002e: 1a                                        ; drop
+000002f: 41                                        ; i32.const
+0000030: 08                                        ; i32 literal
+0000031: 0b                                        ; end
+0000032: 1a                                        ; drop
+0000033: 0b                                        ; end
+0000021: 12                                        ; FIXUP func body size
+000001f: 14                                        ; FIXUP section size
+; move data: [1e, 34) -> [1b, 31)
+; truncate to 49 (0x31)
+;;; STDERR ;;)
+(;; STDOUT ;;;
+
+try-delegate.wasm:	file format wasm 0x1
+
+Code Disassembly:
+
+00001f func[0]:
+ 000020: 06 7f                      | try i32
+ 000022: 06 40                      |   try
+ 000024: 01                         |     nop
+ 000025: 18 01                      |     delegate 1
```

Comment:
looks like delegate should dedent here?

Reply:
Thanks for catching this (& `unwind` too)! Latest commit should fix this issue.